### PR TITLE
feat: activity feed added to loan card

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- Activity feed components added
- Activity feed added to loan card

Screenshot 2024-04-01 at 19 51 30

Screenshot 2024-04-01 at 19 51 47

### CODE REVIEW

File: @kiva/kv-components/tests/unit/specs/components/KvClassicLoanCard.spec.js [hunk 1]
```diff
@@ -133,4 +167,31 @@ describe('KvClassicLoanCard', () => {
 		const countryTagElement = getByText('Uganda');
 		expect(countryTagElement).toBeDefined();
 	});
+	it('activity feed should be showed correctly', async () => {
```
Comment (dyersituations) [inline @kiva/kv-components/tests/unit/specs/components/KvClassicLoanCard.spec.js:170]: Add newline before new test

Comment (dyersituations) [reply @kiva/kv-components/tests/unit/specs/components/KvClassicLoanCard.spec.js:170]: Also doesn't look like `async` is needed.

File: @kiva/kv-components/vue/KvLoanActivities.vue [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+	setup(props) {
+		const {
+			loan,
+			combinedActivities,
+			kvTrackFunction,
+		} = toRefs(props);
+
+		const lightboxOpen = ref(false);
+
+		const modalTitle = computed(() => `Activity for ${loan.value?.name}`);
```
Comment (dyersituations) [inline @kiva/kv-components/vue/KvLoanActivities.vue:175]: Is it possible to add `data-hj-suppress` to the modal title for the borrower name?

Comment (roger-in-kiva) [reply @kiva/kv-components/vue/KvLoanActivities.vue:175]: We can do that adding a title slot to the KvLightbox used here...

**Review by dyersituations — APPROVED** [review]

**Pull request merged**